scripts/watch/config.js

```diff
@@ -0,0 +1,70 @@
+const path = require('path');
+
+const watchDir = path.join(__dirname, '..', '..');
+
+const watchIgnoreTerms = [
+  'node_modules',
+  '.git',
+  '.turbo',
+  'apps/desktop/release',
+  'dist',
+  'coverage',
+  'submodules',
+  'screenshots',
+  'tests',
+  'test-results',
+  'playwright',
+  '__fixtures__',
+  '__helpers__',
+  'e2e',
+  'scripts',
+  'docs',
+];
+
+const watchIgnoreExtensions = ['.log', '.txt'];
+
+const watchFilter = file => {
+  if (watchIgnoreTerms.some(term => file.includes(term))) {
+    return false;
+  }
+
+  if (watchIgnoreExtensions.some(term => file.endsWith(term))) {
+    return false;
+  }
+
+  return true;
```

Comment:
```suggestion
  const containsIgnoreTerms = watchIgnoreTerms.some(term => file.includes(term));

  const containsIgnoreExtensions = watchIgnoreExtensions.some(ext => file.endsWith(ext));

  return !containsIgnoreTerms && !containsIgnoreExtensions;
```

Could be refactored to avoid if conditions